src/ir/LocalGraph.cpp

```diff
@@ -61,6 +61,14 @@ struct Info {
   }
 };
 
+// Minimalist block information needed
+struct OptimizedBlock {
```

Comment:
Maybe `FlowBlock`? Seems like this is only used in `flow()`. Can also move the definition into that function, so the scope makes it clear where it is used.

Reply:
Ok